pkg/apis/sealed-secrets/v1alpha1/sealedsecret_expansion.go

```diff
@@ -254,18 +254,27 @@ func (s *SealedSecret) Unseal(codecs runtimeserializer.CodecFactory, privKeys ma
 		secret.Type = s.Spec.Template.Type
 
 		secret.Data = map[string][]byte{}
+
+		var (
+			badKeys        []string
+			unsealingError error
+		)
 		for key, value := range s.Spec.EncryptedData {
 			valueBytes, err := base64.StdEncoding.DecodeString(value)
 			if err != nil {
 				return nil, err
 			}
 			plaintext, err := crypto.HybridDecrypt(rand.Reader, privKeys, valueBytes, label)
 			if err != nil {
-				return nil, err
+				badKeys = append(badKeys, key)
+				unsealingError = err
 			}
 			secret.Data[key] = plaintext
 		}
 
+		if unsealingError != nil {
+			return nil, fmt.Errorf("%v (affected encrypted data items: %q)", unsealingError, badKeys)
```

Comment:
We should clarify that `%v` was the last observed unsealing error (rather than the "common/same" error on all `badKeys`)